NEWS.md

```diff
@@ -11,6 +11,10 @@
   name with a null character (like `find -print0`).  If one or more -v
   arguments are specified, multiple null characters are used to separate
   fields; see the man page for details.
+- tarsnap now accepts --hashes, which causes --list-archives to prints archive
+  hashes.  If one or more -v arguments are specified, it will print other
```

Comment:
Maybe "archive name hashes" to avoid making people think they can use this to verify the contents of an archive?

Reply:
How about "hashes of archive names"?

Reply:
Sure.